spec/classes/foreman_plugin_openscap_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
+
+describe 'foreman::plugin::openscap' do
+  on_supported_os.each do |os, facts|
+    if facts[:osfamily] == 'RedHat'
+      context "on #{os}" do
+        let :facts do
```

Comment:
@ares why don't you assign facts just after `on_supported_os.each do |os, facts|`?

```puppet
  on_supported_os.each do |os, facts|
    context "on #{os}" do
      let(:facts) do
        facts
      end

      if facts[:osfamily] == 'RedHat'
        ...
      end
   end
```